README.md

```diff
@@ -1169,6 +1170,70 @@ Content-Disposition: form-data; name="workflowOptions"; filename="options.json"
 --f3fd038395644de596c460257626edd7--
 ```
 
+## POST /api/workflows/:version/validate
+
+This endpoint allows WDL to be validated in the context of a running cromwell server and a given inputs file.
```

Comment:
cromwell -> Cromwell

Is it also worth mentioning the kinds of things that Cromwell will validate?  Like, will it validate that the inputs are compatible with a backend?  That they're compatible with their corresponding types in the WDL file?